src/main/kotlin/pet/project/app/dto/book/RequestBookDto.kt

```diff
@@ -0,0 +1,9 @@
+package pet.project.app.dto.book
+
+data class RequestBookDto(
+    val title: String,
+    val description: String?,
+    val yearOfPublishing: Int?,
+    val amazonPrice: Double,
+    val amountAvailable: Int
```

Comment:
```suggestion
    val amountAvailable: Int,
```

It's best practice to add trailing comma

See https://www.baeldung.com/kotlin/trailing-comma-support#the-benefits-of-the-trailing-comma-support

Reply:
Fixed: I investigated the subject and improved the code.